var/spack/repos/builtin/packages/embree/package.py

```diff
@@ -17,6 +17,7 @@ class Embree(CMakePackage):
     depends_on('cmake@2.8.11:', type='build')
     depends_on('ispc', type='build')
     depends_on('ninja', type='build')
+    depends_on('tbb')
```

Comment:
Should `tbb` is variant so that we can enable/disable TBB tasking model?  In case someone want to use embree without multi-threading (I don't know use cases though)

Reply:
It's mainly to force a more recent version provided by spack rather than choosing tbb from the system. They do have an 'internal' tasking system, not sure that's any useful though.